### ISSUE TITLE
Compile fails on Ubuntu 18.04

### ISSUE THREAD

**Issue by Pouya-moh** [vectorgraphics/asymptote#115]
On branch master after configuration I make try to make asymptote upon which the following error happens:

```
In file included from drawelement.h:21:0,
                 from drawpath.h:11,
                 from drawpath.cc:12:
glrender.h:100:8: error: ‘GLuint’ does not name a type; did you mean ‘uint’?
 extern GLuint ubo;
        ^~~~~~
        uint
glrender.h:148:8: error: ‘GLint’ does not name a type; did you mean ‘uint’?
 extern GLint materialShader;
        ^~~~~
        uint
glrender.h:149:8: error: ‘GLint’ does not name a type; did you mean ‘uint’?
 extern GLint colorShader;
        ^~~~~
        uint
glrender.h:150:8: error: ‘GLint’ does not name a type; did you mean ‘uint’?
 extern GLint noNormalShader;
        ^~~~~
        uint
glrender.h:151:8: error: ‘GLint’ does not name a type; did you mean ‘uint’?
 extern GLint pixelShader;
        ^~~~~
        uint

```

Any ideas?

**Comment by johncbowman** [comment]
If you check your configure log you will probably see that one of your OpenGL dependencies is missing (perhaps glm-devel). Once you satisfy that, the code in the master branch should compile.

We are almost ready to make a new release that features WebGL output (for mobile and desktop browsers) and will ensure beforehand that the code compiles again without OpenGL support like before (not recommended, however, as you will only get poor-quality flattened 3D graphics).

**Comment by Pouya-moh** [comment]
@johncbowman Thank you very much for the reply.

> and will ensure beforehand that the code compiles again without OpenGL support like before

Actually before opening the issue, either the autogen or the configurator (cannot recall which one) complained about missing `glm` which I installed. In that regards it did what you wanted. The problem was it didn't find my `glut` but it also didn't complain until the compile time. Only by looking at the config.log (thanks for the hint) I was able to identify the issue.

Cheers!
